mlflow/ml_package_versions.py

```diff
@@ -266,5 +266,18 @@
             "minimum": "4.4.6",
             "maximum": "5.1.8"
         }
+    },
+    "keras": {
+        "package_info": {
+            "pip_release": "keras"
+        },
+        "models": {
+            "minimum": "3.0.2",
+            "maximum": "3.10.0", # Dummy version number to remove cap.
```

Comment:
We can use the latest version. We don't know if the current implementation really works with 3.10.0.

Reply:
sgtm!